components/static.html

```diff
@@ -68,4 +86,4 @@
           </div>
         </noscript>
         <!--loading for realz-->
-        Loading...</div>
+        Loading...</div>
```

Comment:
Probably should keep the new line at end of file. Editor not honoring [`.editorconfig`](https://github.com/uPortal-Project/uportal-app-framework/blob/master/.editorconfig#L28)?